README.md

```diff
@@ -4,6 +4,8 @@
 # PreREISE
 This package gathers and builds demand, hydro, solar, and wind profiles. The
 profiles are needed to run scenarios on the U.S. electrical grid.
+This package is part of a set of packages representing Breakthrough Energy's energy system model.
```

Comment:
To avoid repeating the word `package` and `energy` I would write: `This package along with other ones constitute the Breakthrough Energy's power system model.`

Reply:
Thanks for the review. I took your point into consideration.